ts/project.ts

```diff
@@ -163,58 +179,80 @@ const proxyEnvironmentHelpers = (): ProjectHelpers => ({
     return this.defaultEnvironment.build(s, ...args);
   },
 
-  addExecutable<T extends Project, Name extends string>(this: T, name: Name) {
+  addExecutable<T extends Project, Name extends string>(
+    this: T,
+    name: Name,
+    script?: string,
+  ) {
     if (this.defaultEnvironment == null) {
       throw new Error(
         `'.addExecutable' can only be called on projects with a default environment`,
       );
     }
-    const templateLiteralFn = (
-      s: TemplateStringsArray,
-      ...args: Array<string>
-    ) => {
-      const newExecutable = { [name]: this.shell(s, ...args) } as {
-        [n in Name]: Executable;
-      };
+    if (script != null) {
+      const newExecutable = { [name]: this.shell(script) };
       return {
         ...this,
         ...newExecutable,
```

Comment:
Interesting that you didn't run into the same `name` becoming `string` type degradation that i ran into with the extra function layer. Since this is the case, should we just inline it here?

```suggestion
        [name]: this.shell(script),
```

Reply:
I think this might be because we updated the `deno` version in between?

Inlined in 4b19957c.